FIPS_data.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
```

Comment:
This should also be a part of the package as a workflow function and the variable names should meet PEP8 standards for all lower case and underscore separated.

Reply:
this actually is an old iteratiation and can be deleted. I will remove it from the package